hubstorage/jobq.py

```diff
@@ -31,6 +31,13 @@ def push(self, spider, **jobparams):
                 raise DuplicateJobError()
             raise
 
+    def jobsummary(self, jobkeys, jobmeta):
+        """Fetch selected job metadata fields for selected jobs."""
+        if not isinstance(jobkeys, (list, tuple)):
+            raise TypeError("jobkeys must be a list or a tuple")
+        return list(self.apiget(('jobsummary',),
```

Comment:
I understand it's kinda nitpicking, but it's nice to have consistency across interfaces in open source libraries. `Special cases aren't special enough to break the rules`. I don't think we have any practical reason to break a rule here, do we?